### ISSUE TITLE
Pokemon picross froze

### ISSUE THREAD

**Issue by Enry123** [meladroit/svdt#20]
Hi, I tried to dump my save of Pokemon Picross with svdt several times but svdt just froze!

**Comment by suloku** [comment]
Games released after a certain date (I don't know which) can't be dumped.
I think it is because a different encryption is used, but I don't really know.
Supposedly this might be fixed when smealum updates ninjhax.

In any case, this is not related to svdt, but rather it is a current limitation of ninjhax 2.5.

**Comment by pkmnlover** [comment]
im trying to do this with picross

**Comment by suloku** [comment]
Still no solution for userland, ninjhax 2.6 doesn't adress this issue. This is not a problem related to svdt.

**Comment by juyingnan** [comment]
so pokemon vc(yellow/red/blue)'s issue is because of same reason?

**Comment by suloku** [comment]
Yes. Smea is working on 9.6+ support for ninhax 2.7, so expect the games to work when that gets released

**Comment by juyingnan** [comment]
Thank you. I just tried a latest fork from njnjhax 2.7 (http://ihaveamac.github.io/ninjhax2.x/) and that problem has been fixed (svdt works, but hans doesn't. formal 2.7 should have fixed that.).
Pokemon Picross works and Pokemon VC yellow works as well. I tested both of them.
Thank you all developers involved.

**Comment by suloku** [comment]
Oh, that's great news!

**Comment by Rangnarok123** [comment]
Hi, I am on FW 11, no custom firmware. I am trying to use SVDT on freakyhax, and it still froze. Can anyone offer any advice please?

Thank you

**Comment by suloku** [comment]
https://github.com/ihaveamac/9.6-dbgen-xmls/blob/master/README.md
